### ISSUE TITLE
brscan4 using /opt filepath (T10212)

### ISSUE THREAD

**Issue by celticmagic** [getsolus/packages#117]
<div dir="rtl"><b>Joey&nbsp;Riches</b>&nbsp;(<em>&#35;joebonrichie</em>),&nbsp;2022-03-30&nbsp;12:33:32&nbsp;UTC</div><br/>

We should we about moving it to `/usr/share/brscan4`, we generally don't allow for /opt to be used in our packages.

**Comment by malfisya** [comment]
I don't think this is a good first issue as this might require patching and others.
